### PR TITLE
Update ci.yml

### PR DESCRIPTION
**Pull request by jason-fox**
Add Node 14 to unit-test matrix

### CODE REVIEW

File: docs/deprecated.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ A list of deprecated features and the version in which they were deprecated foll
 
 -   Support to NGSIv1.
 -   Support to Node.js v4 in iotagent-json 1.9.0. The use of Node.js v8 is highly recommended.
+-   Support to Node.js v10 in iotagent-json 1.17.0. The use of Node.js v12 is highly recommended.
```
Comment (fgalan) [inline docs/deprecated.md:17]: For the sake of completeness, maybe information about versions between v4 and v10 should be included (it should be easy to know looking at https://github.com/telefonicaid/iotagent-json/releases )... although maybe it doesn't worth the time.

Not sure how it's better...

File: test/unit/ngsiv2/contextRequests/createBidirectionalDevice.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "id": "Second MQTT Device",
   "type": "AnMQTTDevice",
   "location": {
-    "type": "geo:point",
-    "value": "0, 0"    
+    "type": "geo:json",
+    "value": {"type":"Point","coordinates":[0,0]}
```
Comment (fgalan) [inline test/unit/ngsiv2/contextRequests/createBidirectionalDevice.json:6]: Shouldn't be unit test failing before this fix (I mean, in master)?

Comment (jason-fox) [reply test/unit/ngsiv2/contextRequests/createBidirectionalDevice.json:6]: If you re-start the last run it should now fail. The change has occurred in the downstream iotagent-node-lib

"iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
... is now picking up telefonicaid/iotagent-node-lib@a7eb260 commited 20 days ago, but the last CI test run was 23 days ago.

File: docs/deprecated.md (hunk 1 above)
Comment (jason-fox) [reply docs/deprecated.md]: Fixed b1dccdc

File: test/unit/ngsiv2/contextRequests/createBidirectionalDevice.json (hunk 2 above)
Comment (fgalan) [reply test/unit/ngsiv2/contextRequests/createBidirectionalDevice.json:6]: Re-run in master and failed. Everything ok.

NTC

File: docs/deprecated.md [hunk 3]
```diff
@@ -12,8 +12,13 @@ particular:
 
 A list of deprecated features and the version in which they were deprecated follows:
 
--   Support to NGSIv1.
--   Support to Node.js v4 in iotagent-json 1.9.0. The use of Node.js v8 is highly recommended.
+-   Support to NGSI v1.
+-   Support to Node.js v4 in iotagent-json 1.9.0.
+-   Support to Node.js v6 in iotagent-json 1.10.0.
+-   Support to Node.js v8 in iotagent-json 1.14.0.
```
Comment (fgalan) [inline docs/deprecated.md:18]:
```suggestion
-   Support to Node.js v4 in iotagent-json 1.9.0 (finally removed in 1.10.0)
-   Support to Node.js v6 in iotagent-json 1.10.0 (finally removed in 1.11.0)
-   Support to Node.js v8 in iotagent-json 1.14.0 (finally removed in 1.15.0)
```

File: docs/deprecated.md [hunk 4]
```diff
@@ -33,8 +38,10 @@ in the case you want to use old versions:
...
-| --------------------- | ------------------------------------------------- | ----------------------------- |
-| NGSIv1 API            | Not yet defined                                   | Not yet defined               |
-| Support to Node.js v6 | Not yet defined but it will be done by May 2019   | Not yet defined               |
-| Support to Node.js v4 | 1.9.0                                             | December 19th, 2018           |
+| **Removed feature**    | **Last iotagent-json version supporting feature**   | **That version release date** |
+| ---------------------- | --------------------------------------------------- | ----------------------------- |
+| NGSIv1 API             | Not yet defined                                     | Not yet defined               |
+| Support to Node.js v4  | 1.9.0                                               | December 2018                 |
+| Support to Node.js v6  | 1.10.0                                              | May 2019                      |
+| Support to Node.js v8  | 1.14.0                                              | April 2020                    |
```
Comment (fgalan) [inline docs/deprecated.md:46]:
```suggestion
| Support to Node.js v4  | 1.9.0                                               | December 19th, 2018                 |
| Support to Node.js v6  | 1.10.0                                              | May 22th, 2019                      |
| Support to Node.js v8  | 1.14.0                                              | April 7th, 2020                    |
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**